src/components/Header/index.tsx

```diff
@@ -0,0 +1,53 @@
+import './index.scss';
+
+import React from 'react';
+import { GameState } from '../../types';
+import { IconButton } from '@material-ui/core';
+import { ScoreBoardIcon } from '../../icons';
+
+interface HeaderProps {
+    onScoreBoardOpen: () => void;
+    stage: GameState['stage'];
+    round: GameState['roundNumber'];
+    trick: GameState['trickNumber'];
+    trump: GameState['trumpSuit'];
+}
+
+function Header(props: HeaderProps) {
+    const {
+        onScoreBoardOpen,
+        round,
+        stage,
+        trick,
```

Comment:
sometimes you call this trick and other times trickNumber. Might be less confusing if they were consistent?

Reply:
I could change them all to trickNumber? I have been using trick > trickNumber for brevity, but only when there isn't other trick things to make it confusing

Reply:
addressed https://github.com/creisle/quarantined_wizzards/pull/1/commits/110115d5b48a83680bf13fdd560c486cc1988257